go/base/recent_app.go

```diff
@@ -0,0 +1,20 @@
+package base
+
+import (
+	"time"
+)
+
+// AppThrottle is the definition for an app throtting instruction
+// - Ratio: [0..1], 0 == no throttle, 1 == fully throttle
+type RecentApp struct {
+	CheckedAt           time.Time
+	MinutesSinceChecked int64
+}
+
+func NewRecentApp(checkedAt time.Time) *RecentApp {
+	result := &RecentApp{
+		CheckedAt:           checkedAt,
+		MinutesSinceChecked: int64(time.Since(checkedAt).Minutes()),
```

Comment:
I like that you provided both `CheckedAt` and `MinutesSinceChecked` in the RecentApp struct, because it makes the responses more readable for humans. 

However I lean (generically) towards providing epochs over Time objects, when the data is processed by other computers, as they are monotonic.